browser/views.py

```diff
@@ -687,3 +686,47 @@ def delete_investigator_record(request, person_id, research_id):
         Investigator.objects.filter(id=research_id).delete()
 
     return HttpResponseRedirect(reverse('person', args=(person_id,)))
+
+
+@staff_member_required
+def position(request, person_id, position_id):
+    person = get_object_or_404(Person, pk=person_id)
+    template = "browser/position.html"
+    form = PositionForm()
+    context = {
+        'form': form,
+        'person': person,
+    }
+
+    if position_id is not None:
```

Comment:
```if position_id: ```~~is not None~~

Reply:
Removed ``` is not None ```

Reply:
Can `position_id` be 0? If yes, `if position_id:` might not behave as expected.